assets/src/js/media_archive.js

```diff
@@ -1,6 +1,5 @@
 import ArchivePicker from './Components/ArchivePicker';
-import {render} from '@wordpress/element';
-
+import {createRoot} from '@wordpress/element';
 const rootElement = document.getElementById('archive-picker-root');
 
-render(<ArchivePicker />, rootElement);
+createRoot( rootElement ).render(<ArchivePicker />)
```

Comment:
because of 👇 
```
Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot
```